deployer/src/lib.rs

```diff
@@ -54,12 +55,23 @@ pub async fn start(
     );
     let make_service = router.into_make_service();
 
-    info!("Binding to and listening at address: {}", args.api_address);
+    // todo: don't hardcode this path
+    let runtime_dir = PathBuf::from_str("/home/oddgrd/Documents/shuttle/target/debug").unwrap();
```

Comment:
Can we make this `MANIFEST_PATH` + "target/debug" possibly?

Reply:
Woops, sure thing!